.github/workflows/CI.yml

```diff
@@ -1,30 +1,48 @@
 name: CI
 
 on:
-  push:
-    branches: [ main ]
   pull_request:
-    branches: [ main ]
+    paths-ignore:
+      - "docs/**"
+      - "**/*.md"
+      - ".prettierrc"
+      - "**/*ignore"
+  push:
+    branches:
+      - main
+      - next
+      - rc
```

Comment:
Why do we have a 'next' and 'rc' branch?

Reply:
I am considering introducing them as our formal release pipeline. Essentially you create a `next` branch from an `rc`, and an `rc` on a tag which you believe is ready to be released but needs to be tested in the wild. They imply levels of stability.